tests/prefetch_related/tests.py

```diff
@@ -306,6 +306,28 @@ def test_filter_deferred(self):
             list(Book.objects.prefetch_related('authors'))
             self.assertEqual(add_q_mock.call_count, 1)
 
+    def test_foreignkey_forward_filter_deferred(self):
+        add_q = Query.add_q
+        with mock.patch.object(
+            Query,
+            'add_q',
+            autospec=True,
+            side_effect=lambda self, q: add_q(self, q),
+        ) as add_q_mock:
+            list(Book.objects.prefetch_related('first_book'))
+            self.assertEqual(add_q_mock.call_count, 1)
```

Comment:
`Book` model doesn't have the `first_book` attribute, moreover this test is not related with fix. I think we can chop it.

Reply:
Hey, I meant to use `Author` here but got lost in a copy/paste. I have fixed the reference in d8e75e154a4e441fb0b02c0850f15234f2d3d1b9 and think it's still relevant?

Reply:
It's not related with this patch, because it works without it. Also, as far as I'm aware this is not even related with an optimization added in  681f7e2b13846def9eff3d26ce1371d05fa90c4c, because it works without it. I don't think we need this test. Please remove it.

Reply:
:+1: removed in 31cfd3f1f2dca6b0759230692c33489b5ff61434